### DISCUSSION TITLE
WebSocket Error

### DISCUSSION THREAD

**Discussion by lihe07** [fonsp/Pluto.jl#2584] [Questions about using Pluto] [answered]
I'm using:

- julia `1.9.1`
- pluto `0.19.26`
- HTTP `1.9.6`
- archlinux x64

When I started pluto, my browser reported:

```
WebSocket connection to 'ws://localhost:1234/?secret=FXJZZx9i' failed: Error during WebSocket handshake: Incorrect 'Sec-WebSocket-Accept' header value
```

**Comment by fonsp** [comment]
Can you make a [bug report new issue](https://github.com/fonsp/Pluto.jl/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=), and follow the instructions there to record a video?

**Comment by lihe07** [comment, marked as answer]
The problem disappeared after I updated julia to the latest version.
Anyway, thanks for your efforts on building this amazing software!
